github/util.go

```diff
@@ -12,6 +12,44 @@ const (
 	maxPerPage = 100
 )
 
+func webhookConfigurationSchema() *schema.Schema {
```

Comment:
Don't love where this lives, maybe create a file named something like `webhook_helpers.go`

Reply:
Ah, yeah, I had mixed feelings about that too, let me clean it up 🙂